tests/console_test.go

```diff
@@ -170,33 +170,22 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 		})
 
 		Context("without a serial console", func() {
-			var vmi *v1.VirtualMachineInstance
 
-			BeforeEach(func() {
-				vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+			It("[test_id:4116]should create the vmi without any issue", func() {
```

Comment:
I would probably pick `[test_id:4118]should not connect to the serial console`  as it also implies that vmi is created without issues.